Jakefile

```diff
@@ -476,6 +469,8 @@ desc("Builds the test infrastructure using the built compiler");
 task("tests", ["local", run].concat(libraryTargets));
 
 function exec(cmd, completeHandler) {
+
+return;

```

Comment:
What is this return for? Should we just eliminate this function if we don't need it anymore?